frontend/apps/reader/modules/readerview.lua

```diff
@@ -482,6 +493,12 @@ function ReaderView:drawScrollView(bb, x, y)
         self.state.pos)
 end
 
+--- Converts a color name into a color struct
+function ReaderView:lookupHighlightColor(color_name)
+    local color = self.highlight_colors[color_name]
+    return Blitbuffer.colorFromString(color or "#ffff00")
+end
```

Comment:
"lookup" in the method name feels a bit odd/too generic? @NiLuJe, thoughts? (getBbColor :/?)

Reply:
Sounds fine to me out-of-context; no time today to review in context ;).

Reply:
It's a lookup table. Theoretically it's possible that the saved color name may not map to anything.
I can change it to `getHighlightColor` or whatever people want.

Reply:
`getHighlightColor()` sounds like it would accept a highlight item.
`getColorNameColor()` , `getColorFromColorName()` , `getColorForColorName()`?

Reply:
Updated to `colorFromName`